### DISCUSSION TITLE
In-person and Online Mtg displaying as Online

### DISCUSSION THREAD

**Discussion by aamonterey** [code4recovery/12-step-meeting-list#453] [Q&A] [answered]
When users first land on our /meetings/ page, meetings that should display as ‘In-person and Online’ display as 'Online'. However, if the the filters are used, the information will display properly. 

Too Reproduce:
1) Go to: [https://www.aamonterey.org/meetings/]
2) Note which meetings show as Online
3) Use one of the filters such as selecting a day of week. 
4) Now you'll see meetings correctly listed as In-person and Online

We need to get this information to always display properly. 

Is there a known fix for this issue?

I'm seeing the issue on a Desktop with the following:
 - Windows 10
 - Chrome
 - Version 91.0.4472.124

![AAMonterey Screenshot](https://user-images.githubusercontent.com/87451879/125704123-8f4f758f-5d34-4f2d-bd1c-b2c284f20175.png)

**Comment by tech2serve** [comment, marked as answer]
> 
> Is there a known fix for this issue?
> 
I'm not sure yet. It looks like you've installed v3.11.2; can you confirm that please? Also what version of Wordpress? PHP?

I suspect the fix lies in removing or changing some modifications you've made, at least that is my guess. Your meeting list looks different than ours. Two examples:

Here is your display for the Types dropdown:

<img width="205" alt="image" src="https://user-images.githubusercontent.com/31904316/125830511-5821ce17-8c43-492d-aa42-055a752a98ec.png">

However, in v3.11, you should have the options for online, in-person and active.

Also, the online and in-person flags were moved to display under the Location/Group data. Yours are still showing at the meeting name.

Please look at our [TSML demo](https://demo.code4recovery.org/meetings) to see how the display should look for the meeting list.

Have you added custom types for online meetings so as to not use the conference URL or Phone fields? The single meeting page seems to indicate this. If those fields are not sued, these meetings will likely not show as online in Meeting Guide (per their spec).

v3.11 strives to make significant improvements in how we handle the myriad attendance options. I know groups developed their own approaches (or tweaked our early efforts) to handle online meetings. Our goal is to make those modifications unnecessary and improve data accuracy for Meeting Guide.

My best suggestion is to roll your changes back and let the TSML approach work as designed. This will also help ensure your meetings reflect correctly in Meeting Guide.

**Reply by aamonterey** [reply]
Yes, we're using version 3.11.2,  WP is 5.7.2 and PHP v7.4.

Yes, the online and in-person flags are displaying under Meeting name, but only when first landing on the page. They do display under Location / Group after using filters. 

We have not added any custom types for online meetings so no idea why the _Active, In-person, Online_ doesn't show on our menu. Our only customization is as follows: 
1) A location photo has been added to each meeting page. We prefer not to eliminate this because feedback from our members tells us it's very helpful for newcomers and visitors. Some time ago, the original developer of TSML told me he liked the idea of adding the location photo as a standard feature. Maybe adding this is still under consideration?
2) Custom description for Open and Closed meetings. But I see your demo site is using the exact same description so I expect we can eliminate this customization.

I can have our developer remove customization (2 above, but wanted to provide you a response and get feedback first. 

Here's how the backend looks for one of our meetings. 
![TSML online mtg details](https://user-images.githubusercontent.com/87451879/125870023-84f98f77-718f-4c1f-81b5-f1952a760528.PNG)

**Reply by tech2serve** [reply]
Something is catching up your initial meeting list page, in particular, the `archives-meeting.php` page. And I doubt the images on the `single-meeting.php` page are creating that problem. But, I also don't see the expected Zoom link buttons on that page which is odd.

Are there any error messages in the dev console (research how to open that for your particular browser) when that meeting page loads?

Figure out where your website cache is managed (usually at the hosting provider) and get that cleared.

Also, try disabling other plugins to see if the initial meeting list page starts to load correctly. If your hosting provider includes a caching plugin, that may have prevented the server files from updating. 

If all of these steps fail, you may want to reinstall the plugin. Make sure you have backups and csv exports of your meetings.

**Reply by brianw-area24** [reply]
I think I was able to recreate the issue. I think @tech2serve is correct in that it appears there have been some modifications on your local site, and I think I might know where to look for them. 

You will need access to see/remove files on your file system. How you do this will very depending on your hosting service. They may have give you this access through a terminal window, or on option in cPanel, or whatever. Worst case, you'll have to ask them to remove some files for you.

Please check in the root of your theme's folder, usually located at wp-content/themes/<nameOfTheme>, for example, on my test site, it's the folder wp-content/themes/twentytwenty. You're looking for files with the names: archive-meetings.php, single-locations.php, and single-meetings.php

If you find any of these files, you can rename them, replacing the php extension with bak. After a while running like this, you're probably OK to delete these "bak" files.

Hopefully this will help you out.
Brian W

**Reply by aamonterey** [reply]
We took your suggestions to our developer and we've got everything working - Thank you!! MikeV
